Musicplayer.php

```diff
@@ -1,48 +1,81 @@
+<?php
```

Comment:
21/9: I add a function that deliver to the login page if they was login by want to access to some link that they should login first.